detectron2/data/samplers/distributed_sampler.py

```diff
@@ -124,7 +124,11 @@ def repeat_factors_from_category_frequency(dataset_dicts, repeat_thresh):
         rep_factors = []
         for dataset_dict in dataset_dicts:
             cat_ids = {ann["category_id"] for ann in dataset_dict["annotations"]}
-            rep_factor = max({category_rep[cat_id] for cat_id in cat_ids})
+
+            if len(cat_ids) > 0:
+                rep_factor = max({category_rep[cat_id] for cat_id in cat_ids})
```

Comment:
this is sufficient
```suggestion
                rep_factor = max({category_rep[cat_id] for cat_id in cat_ids}, default=1.0)
```

Reply:
Yes that is a lot cleaner, will fix it, thanks